client/asset/dcr/native_wallet.go

```diff
@@ -93,6 +93,7 @@ type NativeWallet struct {
 
 // NativeWallet must also satisfy the following interface(s).
 var _ asset.FundsMixer = (*NativeWallet)(nil)
+var _ asset.Authenticator = (*NativeWallet)(nil)
```

Comment:
I made you a new conflict at this line, but this line shouldn't be needed anyway. `NativeWallet` embeds `ExchangeWallet` and we already to an interface check on `ExchangeWallet`.